tsweb/tsweb.go

```diff
@@ -423,3 +423,37 @@ func Error(code int, msg string, err error) HTTPError {
 func VarzHandler(w http.ResponseWriter, r *http.Request) {
 	varz.Handler(w, r)
 }
+
+// AddBrowserHeaders sets various HTTP security headers for browser-facing endpoints.
+//
+// The specific headers:
+// - require HTTPS access (HSTS)
+// - disallow iframe embedding
+// - mitigate MIME confusion attacks
```

Comment:
Does this follow the new formatting in https://go.dev/doc/go1.19#go-doc .... https://go.dev/doc/comment#lists ?

My editor always rewrites my comments on save into the new style but this doesn't look like it. (needs a blank line before the first bullet and them all to be indented?)

Reply:
So, the blank line is not required, but two spaces before a dash are required. Updated